util/stats/timer.go

```diff
@@ -95,6 +95,7 @@ func (t Timers) Swap(i, j int) {
 }
 
 func (s byCreationTimeSorter) Less(i, j int) bool {
+	fmt.Printf("i: %v / j: %v\n", s.Timers[i].created, s.Timers[j])
```

Comment:
Debug code?

Reply:
yes to debug the time.Time issue

Reply:
here is a version that does not rely on time.Time